app/javascript/components/ActionItemCards/index.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './ActionItemCard.css';
+import {
+  faLeaf,
+  faHome,
+  faPen,
+  faFile,
+  faSmile,
+  faCode,
+} from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+// Uses CSS File for now. Need to change.
+class ActionItemCard extends React.Component {
+  constructor(props) {
+    super(props);
+    /*
+    TODO: check props types
+    tempory props rn to test components
+    props:
+    actionItem - Javascripy object
+      title - Assignment title: String
+      description - Short description: String
+      date - Due date of assignment: JavaScript date?
+      category - type of assignment: string??
+      status - status of item: string
+    */
+  }
+
+  // render icon based on category
+  renderIcon() {
+    const AssignmentCategory = this.props.actionItem.category;
+    switch (AssignmentCategory) {
+      case 'house':
+        return (
+          <FontAwesomeIcon
+            className="icon-large"
+            icon={faHome}
+            color="#8E8E8E"
```

Comment:
We should probably put all the colors we plan to use in a common utils file for readability/clarity and just import the file and choose the specific color. We don't have to do it now but we should probably make the switch soon (maybe I could do it at Hackathon)? @wilsonw926 @didvi